realsense2_camera/src/t265_realsense_node.cpp

```diff
@@ -72,7 +72,7 @@ void T265RealsenseNode::calcAndPublishStaticTransform(const stream_index_pair& s
 {
     // Transform base to stream
     tf::Quaternion quaternion_optical;
-    quaternion_optical.setRPY(M_PI / 2, 0.0, -M_PI / 2);
+    quaternion_optical.setRPY(M_PI / 2, 0.0, -M_PI / 2);    //Pose To ROS

```

Comment:
@doronhi should the transform here be: `setRPY(-M_PI / 2, M_PI / 2, 0);` ?

According to the [docs](http://docs.ros.org/lunar/api/tf/html/c++/classtf_1_1Quaternion.html#ab34dfef6a01c1ece07bea92d485b0576) setRPY uses a fixed transform. 

I created a short  Python to test this transform by creating: world->t265_pose->ros_frame transforms and visualizing them in rviz. Here is the resultant transform for setRPY(M_PI / 2, 0.0, -M_PI / 2); equivalent:

![image](https://user-images.githubusercontent.com/6543766/64104492-9b7be800-cd74-11e9-889e-124318ba141f.png)
The frames in order from the left are: world_frame->t265_pose_frame->ros_frame.

Here is the gist of the python script that I used: https://gist.github.com/msadowski/caa15bf42bbcc46979eac1afe9f8876b.